tests/portability/cmsis_rtos_v2/prj.conf

```diff
@@ -14,3 +14,5 @@ CONFIG_CMSIS_V2_MEM_SLAB_MAX_DYNAMIC_SIZE=128
 CONFIG_CMSIS_V2_THREAD_MAX_COUNT=23
 CONFIG_CMSIS_V2_THREAD_DYNAMIC_MAX_COUNT=10
 CONFIG_TIMEOUT_64BIT=n
+CONFIG_CMSIS_V2_THREAD_MAX_STACK_SIZE=1024
```

Comment:
what arches/boards were we overflowing the stack?

Reply:
up squared